governance/reviews/template.md

```diff
@@ -62,28 +87,92 @@ Over the next year, the project should work on the following issues to improve i
-| Communication and Meetings | Complete/Partial/Missing | *LINKS* | |
+<!--- 
+Provide the links in the table to the documents/pages that are relevant to the governance area. The links should be to the permanent addresses to snapshots (commits) of the file under evaluation as a point-in-time reference to this review. To ensure consistency, use same commits as links for multiple files within a single repository.
+--->
+
+<!--- 
+
+Project Purpose: 
+- Is the project explaining its purpose/mission/scope/values/principles properly? 
+- A good example template is https://contribute.cncf.io/maintainers/governance/charter/
+
+Maintainer List:
+- Is there a maintainer list?
+- Does it contain employer affiliations?
+- Does it contain roles and responsibilities? The most basic one should look like the template: https://github.com/cncf/project-template/blob/main/MAINTAINERS.md
+- You do not need to assess things like employer balance here. There is a separate section called "Maintainer List(s)" for that in the document.
+
+Code of Conduct:
+- Is CNCF CoC adopted across the whole project? 
+- Is the process of reporting and handling the violations documented and is it complete?
+
+Contributor Guide:
+- Does the governance mention a contributor guide?
+- Is it fresh? (technical contribution guides shall not be assessed part of the govenrnance review)
+
+Contributor Ladder:
+- Does the governance list the criteria to earn a title in the project? The title may depend on the project (maintainer/lead/approver/contributor/etc.).
+- Are there enough roles, including some intermediate ones?
+- Recommended template: https://github.com/cncf/project-template/blob/main/CONTRIBUTOR_LADDER.md
+
+Maintainer Lifecycle:
+- Does the governance doc define when and how a maintainer can be removed/demoted because of inactivity, voluntary stepping down, code of conduct violations?
+- How about emeritus status?
+- Does the replacement maintainer selection make sense? Is the process documented?
+- How about lifecycle for the other roles? (committee members, leads, ...)
+
+Decision-making:
+- Does the governance doc define who the decision makers are?
+- Is the decision making process documented?
+- Is the decision making process consistent and logical?
+
+Code and Docs Ownership:
+- Does the governance doc define who has write/admin access to the code and docs?
+- Only assess if the ownership is documented and if it makes sense. Auditing the permissions is not in the scope of this section.
+
+Security Reporting and response:
+- Is security reporting and response processes documented?
+- Is it in alignment with the guidelines here at minimum https://contribute.cncf.io/maintainers/templates/governance-maintainer/#security-response-team ?
+
+Communication and Meetings:
+- Is project communication channels and meetings documented about when and where they happen?
+
+--->
+
+| Governance Area | Coverage | Finding Notes |
```

Comment:
The reason for the LINKS column is that in some cases, projects may have multiple documents that cover a particular area.  As such, making the area name a link is not sufficient.

Reply:
This makes sense. Reverting the columns in this table.

Reply:
Reverted